src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IncentiveLevelReview.kt

```diff
@@ -4,7 +4,7 @@ import io.swagger.v3.oas.annotations.media.Schema
 import java.time.LocalDate
 
 @Schema(description = "Incentive review information for a prisoner")
-data class IncentiveReview(
+data class IncentiveLevelReview(
```

Comment:
Why renaming to `IncentiveLevelReview` instead of the simpler `IncentiveReview`? Was it clashing with anything?

Reply:
yes the DTO

Reply:
was same as the JPA

Reply:
Wouldn't they be in separate packages/namespaces and be fine tho?

Reply:
Yes but it will require further refactor, the package names will need to be explicit, however happy to do it

Reply:
Done